### PR TITLE
Added JSON file for DB Configurations

### PR DESCRIPTION
**Pull request by JosiahRegencia**
This is for #36 
It's still a git file But I wanted you see this first whether this is fine for you or not.

### CODE REVIEW

File: savant/app/db_config.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (greenmoss) [inline savant/app/db_config.json]: This is good. However you need to call it `db_config_travis.json`. Use your Travis configuration to copy it to `db_config.json` when the test starts.

**Review by greenmoss — CHANGES_REQUESTED** [review]

File: savant/app/db_config.json (hunk 1 above)
Comment (JosiahRegencia) [reply savant/app/db_config.json]: Does this mean that db_config.json is empty?

**Comment by JosiahRegencia** [comment]
made changes. not sure what you really mean. but the db_config.json is now empty. and I added   - cat savant/app/db_config_travis.json >> savant/app/db_config.json that to travis configuration to copy db_config_travis.json to db_config.json.

**Comment by JosiahRegencia** [comment]
removed the db_config.

**Pull request merged**